Describe how to manage the sales reports (#1469)
magento/merchdocs: src/reports/sales-reports.md

*Braintree Settlement Report*

The [Braintree]({% link payment/braintree.md %}) Settlement Report can be filtered according to creation date, amount, status, transaction type, payment type, transaction ID, order ID, PayPal payment ID, type, merchant account ID or settlement batch ID. The report contains the transaction ID, order ID, PayPal payment ID, type, creation date, amount, settlement code, status, settlement response text, reimbursement IDs, merchant account ID, settlement batch ID, and the currency used.

## How to manage Sales Reports

### Step 1: Refresh the report statistics

1. On the _Admin_ sidebar, go to **Reports**.

1. Choose the reports you want to refresh.

> 1. To update statistics, click on the anchor text `here`.

Reviewer: ```suggestion
1. To update statistics, click **here** in the text.
```

### Step 2: Filter reports

> Admin can get a whole website or one store report, a report for a time interval or a date. To filter sale report set:

Reviewer: ```suggestion
In the Admin, you can access can generate a report for a whole website or for one store, within a time interval or a date. To filter a sale report, set the following options:
```
